assets/css/styles.css

```diff
@@ -373,21 +255,1153 @@ header {
+  align-items: flex-start;
+  padding: 0;
+  position: static;
+  width: 327px;
+  height: 409px;
+  left: 24px;
+  top: 114px;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin: 24px 0;
+}
+
+.about-content-top h3 {
+  position: static;
+  width: 327px;
+  height: 101px;
+  left: 0;
+  top: 0;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 40px;
+  line-height: 52px;
+  display: flex;
+  align-items: center;
+  color: #172b4d;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin: 12px 0;
+}
+
+.about-content-top p {
+  position: static;
+  width: 327px;
+  height: 144px;
+  left: 0;
+  top: 113px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+  margin: 12px 0;
+}
+
+.social-about {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 0;
+  position: static;
+  width: 200px;
+  height: 80px;
+  left: 0;
+  top: 269px;
+  flex: none;
+  order: 2;
+  flex-grow: 0;
+  margin: 12px 0;
+}
+
+.social-text-about {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 0;
+  position: static;
+  width: 117px;
+  height: 24px;
+  left: 0;
+  top: 0;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin: 16px 0;
+}
+
+.social-text-about h2 {
+  position: static;
+  width: 117px;
+  height: 24px;
+  left: 0;
+  top: 0;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 14px;
+  line-height: 24px;
+  color: #7f8cff;
+}
+
+.social-icon-about {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 0;
+  position: static;
+  width: 200px;
+  height: 40px;
+  left: 0;
+  top: 40px;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+  margin: 16px 0;
+}
+
+.twitter-icon-about {
+  position: absolute;
+  left: 20%;
+  top: 500px;
+  bottom: 18.88%;
+}
+
+.github-icon-about {
+  position: absolute;
+  left: 39.13%;
+  top: 500px;
+  bottom: 18.87%;
+}
+
+.linkedin-icon-about {
+  position: absolute;
+  width: 20px;
+  height: 20px;
+  top: 500px;
+  left: 30.7%;
+}
+
+.about-m-icon {
+  position: absolute;
+  left: 10.47%;
+  top: 500px;
+  bottom: 19.21%;
+}
+
+.finger-icon-about {
+  position: absolute;
+  left: 48%;
+  top: 500px;
+  bottom: 18.87%;
+}
+
+.resume {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 12px;
+  position: static;
+  width: 162px;
+  height: 48px;
+  left: 0;
+  top: calc(50% - 48px / 2 + 180.5px);
+  border: 1px solid #6070ff;
+  box-sizing: border-box;
+  border-radius: 8px;
+  flex: none;
+  order: 3;
+  flex-grow: 0;
+  margin: 12px 0;
+}
+
+.resume:hover {
+  background: #6070ff;
+  box-shadow: 0 8px 16px rgba(64, 83, 252, 0.24);
+  border-radius: 8px;
+}
+
+.resume:active {
+  top: calc(50% - 48px / 2 + 57.5px);
+  background: #2230d2;
+  border-radius: 8px;
+}
```

Comment:
-  [ ] Please change the color of the 'Get My Resume' text when hovered or active as defined in the template.

**Currently, when active it looks like this:**
![emma-get-resume](https://user-images.githubusercontent.com/36542986/134599725-bb9232d1-0c1b-4aa1-af6f-806eccb6d51f.jpg)

**But the template has this:**

![btns-styles-2](https://user-images.githubusercontent.com/36542986/134599803-a64982d4-9839-4ea8-bfc0-906af1a610ab.jpg)